playpen/all_26.sh

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/bash -e
+
+
+# You need the .bashrc that the Vagrant or dev-setup.sh environment creates. You can get it from
+# playpen/ansible/roles/dev/files/bashrc if you aren't using one of those envs.
+. ~/.bashrc
+
+# Switch back to pulp-2.6
+pstop
+
+pushd ~/devel
+for r in {pulp_deb,pulp_docker,pulp_openstack,pulp_ostree,pulp_puppet,pulp_python,pulp_rpm,pulp}; do
```

Comment:
I am only using bash for this so I am going to leave it as is. Feel free to add support for other shells in a separate PR.